.ci/e2e-expected/dml-batching.txt

```diff
@@ -0,0 +1 @@
+INSERT 0 0
```

Comment:
Should we return the total number of rows inserted? I think we can get this number from JDBC

Reply:
Added.